lib/runner/cli/cli.js

```diff
@@ -308,6 +308,7 @@ class CliRunner {
   }
 
   createTestRunner() {
+    this.test_settings.original_settings = this.settings;
```

Comment:
what is this for?

Reply:
I am using it here: 
https://github.com/nightwatchjs/nightwatch/pull/3364/files#diff-2563b8ebeafa996571324644ca698630c75d72e978e917f2ded557bab4e05830R309


but there must be a better way, I am trying to refactor this.